### DISCUSSION TITLE
Support for OPDS PS

### DISCUSSION THREAD

**Discussion by driftywinds** [edrlab/thorium-reader#2563] [Ideas]
Can you implement page streaming support of the OPDS PS format? (more info - https://anansi-project.github.io/docs/opds-pse/intro)

**Comment by llemeurfr** [comment]
Hi, I think this is more of a discussion than an issue. I'm making the conversion now.

**Comment by llemeurfr** [comment]
We can think about streaming support in two ways. One is what this OPDS PS extension proposes = add support for direct page access into the OPDS feed. Note the `page` and `width` parameters, very pragmatic but not really flexible (what about publications that are not comics?). 

Another is an alternative access to a Web Publication Manifest in the OPDS feed, i.e. a streamable version of the publication. A reading app that supports Web Publications can therefore open the publication an provide full navigation, metadata display, etc.. I think that the differentiation between cataloguing publications and reading publications is clearer this way.

**Comment by driftywinds** [comment]
Yes! The second option sounds great!
